### PR TITLE
[11.0][ADD] stock_picking_deactivate_immediate_transfer

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
Adds module stock_picking_deactivate_immediate_transfer

### CODE REVIEW

File: stock_picking_deactivate_immediate_transfer/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+{
+    'name': 'Stock Picking Deactivate Immediate Transfer',
+    'version': '11.0.1.0.0',
+    'category': 'Warehouse Management',
+    'summary': "Deactivates immediate transfers"
+               "sticking only to planned ones",
+    'author': "Eficent, "
+              "Odoo Community Association (OCA)",
+    'website': 'http://www.eficent.com',
```
Comment (rousseldenis) [inline stock_picking_deactivate_immediate_transfer/__manifest__.py]: Please change this to https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11

File: stock_picking_deactivate_immediate_transfer/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    'depends': ['stock'],
+    'data': [
+        'views/stock_picking_views.xml',
+    ],
+    'installable': True,
+    'license': "AGPL-3",
+}
```
Comment (rousseldenis) [inline stock_picking_deactivate_immediate_transfer/__manifest__.py:22]: Maybe you could improve it with some missing entries https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Tested on runbot :+1: 
Minor comments

**Review by rousseldenis — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
Functional and technical review :+1: 

Thanks!

**Pull request merged**